src/llmq/quorums.cpp

```diff
@@ -161,7 +162,7 @@ CQuorumManager::CQuorumManager(CEvoDB& _evoDb, CBLSWorker& _blsWorker, CDKGSessi
 
 void CQuorumManager::UpdatedBlockTip(const CBlockIndex* pindexNew, const CBlockIndex* pindexFork, bool fInitialDownload)
 {
-    if (fInitialDownload) {
+    if (!masternodeSync.IsBlockchainSynced()) {
```

Comment:
Does this actually make a difference?

Reply:
`fInitialDownload` switches to `false` ~144 blocks deep https://github.com/dashpay/dash/blob/master/src/validation.h#L129 which is ~6 llmq50 quorums. I assume it could be pretty wasteful to scan/ensure connections for these quorums (which are going to be dropped soon after sync is done) but I haven't tested this tbh.

Reply:
Ah yeah, noticed this wasteful building-up of connections as well :+1: